metricbeat/module/apache/status/status.go

```diff
@@ -32,22 +32,19 @@ var (
 )
 
 func init() {
-	if err := helper.Registry.AddMetricSeter("apache", "status", New); err != nil {
+	if err := mb.Registry.AddMetricSet("apache", "status", New); err != nil {
 		panic(err)
 	}
 }
 
-// New creates new instance of MetricSeter
-func New() helper.MetricSeter {
-	return &MetricSeter{}
-}
-
-type MetricSeter struct {
+// MetricSet for fetching Apache HTTPD server status.
+type MetricSet struct {
+	mb.BaseMetricSet
 	URLs map[string]string // Map of host to endpoint URL.
 }
 
-// Setup the URLs to the mod_status endpoints.
-func (m *MetricSeter) Setup(ms *helper.MetricSet) error {
+// New creates new instance of MetricSeter
+func New(base mb.BaseMetricSet) (mb.MetricSet, error) {
```

Comment:
Looks like I did too, that function documentation refers to `MetricSeter`. Will fix.